src/System.Text.Json/tests/JsonElementWithNodeParentTests.cs

```diff
@@ -122,8 +122,18 @@ public static void TestString()
         [Fact]
         public static void TestBytesFromBase64()
```

Comment:
Can you share the test coverage report for this PR? We are trying to keep coverage close to 100% where possible.

Reply:
Absolutely. I'll share it when I've committed the changes for all suggestions.

Reply:
For some reason, I couldn't attach the zip-file to this comment but was able to in a separate comment [here](https://github.com/dotnet/corefx/pull/41467#issuecomment-539978884).